sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, **kwargs):
         self.custom_endpoint_hostname = None
         self.hostname = kwargs.pop("hostname")
         uamqp_transport = kwargs.pop("uamqp_transport")
+        self.amqp_library = PYAMQP_TRANSPORT if not uamqp_transport else UAMQP_TRANSPORT
```

Comment:
is the `self.amqp_library` used anywhere else? (i.e. reason for making this an instance var?)

if not, maybe we can make this `amqp_user_agent_prefix`/`amqp_library`/etc.

Reply:
Ill make them _LIBRARY given thats the variable name I use to store em